group_vars/all.yml.dist

```diff
@@ -181,6 +182,15 @@ samba_shares:
     browsable: yes
     path: "{{ samba_shares_root }}/photos"
 
+###
+### Time Machine
+###
+timemachine_data_directory: /mnt/Volume3/timemachine # needs to be an absolute path
```

Comment:
For consistency, please change `timemachine_data_directory` to  `{{ docker_home }}/timemachine` by default.

Reply:
Done.